### DISCUSSION TITLE
Azuracast does not automatically put a news.mp3 file in the playlist

### DISCUSSION THREAD

**Discussion by FD20201** [AzuraCast/AzuraCast#4192] [Q&A]
Hey Guys, 
Our Radionews comes from an Script so this downloads and copies a news.mp3 every hour to the News folder of Azuracast.
We want that Azuracast is setting the file automatically into theNews playlist.
The Script removes the old file named news.mp3 and downloads the new file like news.mp3.
Can anybody help

**Comment by BusterNeece** [comment]
@FD20201 Have you tried putting it in a folder, and then assigning the _folder_ to the playlist? That should result in any newly added files in that folder being added to the playlist automatically within a few minutes of being uploaded.
